peachjam/templates/peachjam/_taxonomies.html

```diff
@@ -1,15 +1,20 @@
 {% load peachjam i18n %}
 
-<div class="col-md">
-  <h5>{% translate 'Taxonomies' %}</h5>
-  {% for taxonomy in taxonomies %}
+<h5>{% translate 'Taxonomies' %}</h5>
+{% for taxonomy in taxonomies %}
+  <div class="d-flex pt-2">
     {% for item in taxonomy.topic.get_ancestors %}
-      <a href="{% build_taxonomy_url item %}">
-        {{ item }}
-      </a> &gt;
+      <div class="me-2 d-flex align-items-center">
```

Comment:
IMHO this is over-engineered. Start with simple, and if that's sufficient, stop there. Reserve the creativity for elsewhere.

Reply:
refactored